libbulletc/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ ELSE()
     SET(BULLETC_LIB libbulletc)
 ENDIF()
 
+OPTION(USE_DOUBLE_PRECISION "" ON)

```

Comment:
In this branch, the .NET part of the library uses single precision, so this option should be off. There is a separate branch for [double precision](https://github.com/AndresTraks/BulletSharpPInvoke/tree/double-precision).

I haven't found an easy way to make the C# project support both float and double in the same code.
Single precision is still suitable for most scenarios, so I'll keep that as the default, like it is in Bullet.

Reply:
The rest looks good, so if you remove this line, I'll merge the request.
Thanks!

Reply:
I removed it.